packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -68,17 +68,18 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
   let account2Address: string;
   let expectedGasPrice: string;
 
-  const CHAIN_ID = ConfigService.get('CHAIN_ID') || '0x12a';
+  const CHAIN_ID = (ConfigService.get('CHAIN_ID') as string) || '0x12a';
```

Comment:
is there a bug that's being fixed here?
It seems right just curious if it's impacting any tests

Reply:
This just removes a warning in the IDE.

Reply:
Done.

Reply:
Just FYI these are known issues and capture in this ticket https://github.com/hashgraph/hedera-json-rpc-relay/issues/3142